library/block/block_content_registry/src/main/java/org/quiltmc/qsl/block/content/registry/api/BlockContentRegistries.java

```diff
@@ -129,5 +131,17 @@ public class BlockContentRegistries {
 					Integer.class,
 					Codec.intRange(1, 15))
 			.build();
+
+	/**
+	 * A {@link RegistryEntryAttachment} for enchantment power levels in bookshelf equivalents.
+	 * <p>
+	 * Values can be set via code and through a data-pack with the file {@code data/quilt/attachments/minecraft/block/enchantment_boosters.json}
+	 */
+	public static final RegistryEntryAttachment<Block, Float> ENCHANTMENT_BOOSTERS = RegistryEntryAttachment
+			.builder(Registry.BLOCK,
+					new Identifier(QUILT_NAMESPACE, "enchantment_boosters"),
+					Float.class,
+					Codec.floatRange(0f, Float.MAX_VALUE))
```

Comment:
I'm not entirely familiar with how this works, but the last parameter to ENCHANTMENT_BOOSTERS.put(...) would be the amount of equivalent bookshelves to add, correct? And is this line restricting that amount to positive values only?

If so, it might be nice to allow negative values as well. As for why someone might want to ***reduce*** the power level of the enchantment table, that would require some creativity from the modder, but I think it could have valid uses. 

For instance, maybe someone might add a special enchanting block that has both benefits and drawbacks, like something that adds mending as a possible enchantment but reduces the enchanting level by 8. Or perhaps another mod might overhaul enchanting such that having low enchanting power is useful as well. Or... maybe even something that gives different enchants for positive vs negative enchanting power (like a white arts/dark arts dichotomy)